src/java/us/kbase/mobu/initializer/ModuleInitializer.java

```diff
@@ -219,12 +217,31 @@ public void initialize(boolean example) throws Exception {
 			fillTemplate(moduleContext, templateName, templateFiles.get(templateName));
 		}
 		
+		if (example) {
+			// Generated examples require some other SDK dependencies
+			List <String> requiredDependantModules = Arrays.asList("KBaseReport","AssemblyUtil");
+			ClientInstaller clientInstaller = new ClientInstaller(new File(moduleDir), false);
+			for(String dependantModuleName : requiredDependantModules) {
+				clientInstaller.install(
+						this.language,
+						true, // async clients
+						false, // core or sync clients
+						false, // dynamic client
+						"release", //tagVer
+						this.verbose,
+						dependantModuleName,
+						null,
+						null // clientName
+					);
+			}
+		}
+
 		System.out.println("Done! Your module is available in the " + moduleDir + " directory.");
 		if (example) {
 			System.out.println("Compile and run the example methods with the following inputs:");
 			System.out.println("  cd " + moduleDir);
-			System.out.println("  make          (could be necessary after changes in " + new File(specFile).getName() + ")");
-            System.out.println("  kb-sdk test   (will require to set GlobusOnline test account in test_local/test.cfg)");
+			System.out.println("  make          (required after making changes to " + new File(specFile).getName() + ")");
```

Comment:
If a new function is added, there's no stub in the Impl to add code until you run make- although yes, other changes wouldn't necessarily require a make.